src/app/teacher/milestone/milestone-report-graph/milestone-report-graph.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { UtilService } from '../../../../assets/wise5/services/utilService';
+import * as Highcharts from 'highcharts';
+
+@Component({
+  selector: 'milestone-report-graph',
+  template:
+    '<highcharts-chart [Highcharts]="Highcharts" [options]="chartConfig"></highcharts-chart>'
+})
+export class MilestoneReportGraphComponent implements OnInit {
+  DEFAULT_COLOR = 'rgb(194, 24, 91)';
+
+  @Input() barColor: string;
+  categories: any[] = [];
+  chartConfig: any;
+  @Input() data: any;
+  Highcharts: typeof Highcharts = Highcharts;
+  @Input() id: string;
+  @Input() locations: any[];
+  @Input() name: string;
+  series: any[];
+  @Input() titleColor: string;
+
+  constructor(private utilService: UtilService) {}
+
+  ngOnInit(): void {
+    this.data = JSON.parse(this.data.replace(/\'/g, '"'));
+    if (this.name == null) {
+      this.name = this.id;
+    }
+    this.calculateGraphDataAndCategories();
+    this.setConfig();
+  }
+
+  private setConfig(): void {
+    this.chartConfig = {
+      chart: {
+        type: 'column',
+        height: 200,
+        width: 200,
+        style: {
+          fontFamily: 'Roboto,Helvetica Neue,sans-serif'
+        }
+      },
+      title: {
+        text: this.name,
+        style: {
+          fontSize: '14px',
+          fontWeight: '500',
+          color: this.titleColor ? this.titleColor : this.DEFAULT_COLOR
+        }
+      },
+      plotOptions: {
+        series: {
+          dataLabels: {
+            enabled: true,
+            format: '{y}%'
+          }
+        },
+        column: {
+          dataLabels: {
+            style: {
+              fontSize: '10px'
+            }
+          }
+        }
+      },
+      tooltip: {
+        formatter: function () {
+          return `<b>${this.series.name}<br/>${$localize`teams`}: ${this.point.count}</b>`;
+        }
+      },
+      xAxis: {
+        categories: this.categories
+      },
+      yAxis: {
+        title: {
+          text: ''
+        },
+        labels: {
+          enabled: false
+        }
+      },
+      series: this.series
```

Comment:
We can hide the Download image button and `Highcharts.com` link at the bottom right of the graph if we add this to the chartConfig.

```
      exporting: {
        enabled: false
      },
      credits: {
        enabled: false
      },
      series: this.series
```

Reply:
Added. Thanks for the tip.